### DISCUSSION TITLE
Catalyst Feature Vote for Catalyst v4.0.0

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#460] [Polls]
Select a feature or make a suggestion in the comments.

**Comment by ajskateboarder** [comment]
Support for installing browser extensions (likely Chromium-based ones because of WebView) would be great, I'm sure most who use this browser would like a download manager

**Reply by jdev082** [reply]
> Support for installing browser extensions (likely Chromium-based ones because of WebView) would be great, I'm sure most who use this browser would like a download manager

I don't think you understand how difficult that would be to implement. If you want that feature to have any chance of existing someone else is going to have to implement it.

**Reply by ajskateboarder** [reply]
[Electron supports it](https://www.electronjs.org/docs/latest/api/extensions)

**Reply by jdev082** [reply]
Integrating it with the UI, and writing code to make over 25% of what an extension can do work isn't an easy task.

**Reply by jdev082** [reply]
Also not a fan of Manifest v3

**Comment by jdev082** [comment]
It doesn't seem like implementing a download manager is possible, so I have removed it from the list.
